Gemfile.lock

```diff
@@ -557,7 +557,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.0.0)
     raindrops (0.13.0)
-    rake (10.4.2)
+    rake (10.5.0)
```

Comment:
The official changes between 10.4.2 and 10.5 look pretty safe to me:

* [Rake History.rdoc](https://github.com/ruby/rake/blob/2ffe31d0dfb59ef00f3af5f1c27fe33153c01532/History.rdoc)
* [10.4.2 to 10.5.0 comparison](https://github.com/ruby/rake/compare/v10.4.2...v10.5.0)

If you can still run rake without any errors then I'd say it's fine e.g. try `bundle exec rake routes`.

What I do recommend is, to split the two concerns here. Split the change to put the rake update in a separate commit, that way if someone wants/needs to revert that ever it can be done with `git revert` and it makes it very visible in the log when the change happened.

My 2p.